client/core/core.go

```diff
@@ -2392,6 +2392,47 @@ func (c *Core) handleReconnect(host string) {
 		log.Errorf("unable to authorize DEX at %s: %v", host, err)
 		return
 	}
+
+	dc.marketMtx.RLock()
+	for mktKey, mkt := range dc.marketMap {
+		if !mkt.Subscribed() {
```

Comment:
Just loop through the `(*dexConnection).books` map to see what subscriptions are open.